docs/guides/index.mdx

```diff
@@ -381,7 +381,7 @@ This returns the `playbackURL` at the bottom of the response which we will use t
 ##  Playback a Livestream
 
 The response from requesting an asset in the API will return the `playbackId` which is used for the `playbackUrl`. https://livepeercdn.com/hls/{playbackId}/index.m3u8
```

Comment:
https://livepeercdn.com/hls/{playbackId}/index.m3u8 not being a code snippet seems a bit odd -- should it be in a code snippet or a code block?